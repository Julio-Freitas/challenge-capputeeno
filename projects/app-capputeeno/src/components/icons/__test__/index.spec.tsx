import { DeleteIcon } from '../delete-icon';
import { GitHubIcon } from '../github-icon';
import { renderWithProvider } from '@/utils/tests/helpers';

describe('<Gihub', () => {
    it('should render github icon', () => {
        const { container } = renderWithProvider(<GitHubIcon />);
        expect(container.firstElementChild).toMatchSnapshot();
    });
    it('should render delete icon', () => {
        const { container } = renderWithProvider(<DeleteIcon />);
        expect(container.firstElementChild).toMatchSnapshot();
    });
});
