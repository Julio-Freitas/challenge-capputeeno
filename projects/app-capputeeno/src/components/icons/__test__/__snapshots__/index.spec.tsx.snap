// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Gihub should render delete icon 1`] = `
<svg
  fill="none"
  height="24"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    clip-rule="evenodd"
    d="M16.1378 21H7.85782C6.81082 21 5.94082 20.192 5.86282 19.147L4.96582 7H18.9998L18.1328 19.142C18.0578 20.189 17.1868 21 16.1378 21V21Z"
    fill-rule="evenodd"
    stroke="#DE3838"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="1.5"
  />
  <path
    d="M12 11V17"
    stroke="#DE3838"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="1.5"
  />
  <path
    d="M4 7H20"
    stroke="#DE3838"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="1.5"
  />
  <path
    d="M17 7L15.987 4.298C15.694 3.517 14.948 3 14.114 3H9.886C9.052 3 8.306 3.517 8.013 4.298L7 7"
    stroke="#DE3838"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="1.5"
  />
  <path
    d="M15.4298 11L14.9998 17"
    stroke="#DE3838"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="1.5"
  />
  <path
    d="M8.57016 11L9.00016 17"
    stroke="#DE3838"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="1.5"
  />
</svg>
`;

exports[`<Gihub should render github icon 1`] = `
<svg
  aria-hidden="true"
  data-testid="GitHubIcon"
  fill="#737380"
  focusable="false"
  height="24"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M12 1.27a11 11 0 00-3.48 21.46c.55.09.73-.28.73-.55v-1.84c-3.03.64-3.67-1.46-3.67-1.46-.55-1.29-1.28-1.65-1.28-1.65-.92-.65.1-.65.1-.65 1.1 0 1.73 1.1 1.73 1.1.92 1.65 2.57 1.2 3.21.92a2 2 0 01.64-1.47c-2.47-.27-5.04-1.19-5.04-5.5 0-1.1.46-2.1 1.2-2.84a3.76 3.76 0 010-2.93s.91-.28 3.11 1.1c1.8-.49 3.7-.49 5.5 0 2.1-1.38 3.02-1.1 3.02-1.1a3.76 3.76 0 010 2.93c.83.74 1.2 1.74 1.2 2.94 0 4.21-2.57 5.13-5.04 5.4.45.37.82.92.82 2.02v3.03c0 .27.1.64.73.55A11 11 0 0012 1.27"
  />
</svg>
`;
